boards/cc2538dk/include/periph_conf.h

```diff
@@ -59,33 +59,29 @@ static const timer_conf_t timer_config[] = {
 /** @} */
 
 /**
- * @name UART configuration
+ * @name    UART configuration
  * @{
  */
-#define UART_NUMOF          1
+static const uart_conf_t uart_config[] = {
+    {    /* Virtual COM Port */
+        .dev      = (cc2538_uart_t *)0x4000c000,
+        .rx_pin   = GPIO_PIN(0,0),
+        .tx_pin   = GPIO_PIN(0,1),
+        .irq      = UART0_IRQn,
+        .rxd      = UART0_RXD,
+        .txd      = UART0_TXD,
+        .cts_pin  = GPIO_UNDEF,
+        .rts_pin  = GPIO_UNDEF
+    }
+};
 
-#define UART_0_EN           1
-#define UART_1_EN           0
-#define UART_2_EN           0
-#define UART_3_EN           0
+/* interrupt function name mapping */
+#define UART_0_ISR          isr_uart0
 
+/* macros common across all UARTs */
 #define UART_IRQ_PRIO       1
```

Comment:
no need for this value here: all IRQs are set to prio 1 per default in the cortexm initialization.